app/views/layouts/nav/_notifications_link.html.haml

```diff
@@ -1,4 +1,25 @@
-= link_to notifications_path do
-  %i.fa.fa-envelope
+%li{class: "notifications-link dropdown"}
   - if current_user.notifications.unread.length > 0
```

Comment:
I've noticed in several places that this partial is pretty smart about the internals of notifications.  If we decide that 10 is not the right cut off point for displaying, we have several places to change.  What about naming the concepts needed here and pulling them into Notification.rb?  That way the partial only needs to use the methods that expose those concepts.